file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,322 @@
+      oauth_profile         TYPE c LENGTH 30,
+    END OF ty_intelligent_scenario,
+
+    "! <p class="shorttext">Binding Mappings</p>
+    "! These are scenario binding mappings
+    BEGIN OF ty_binding_mappings,
+      "! <p class="shorttext">Signature Name</p>
+      "! This is signature name
+      signature_name TYPE c LENGTH 30,
+      "! <p class="shorttext">Table or View Field Name</p>
+      "! This is a table or view field name
+      from_field     TYPE c LENGTH 30,
+      "! <p class="shorttext">Table or View Field Name</p>
+      "! This is a table or view field name
+      to_field       TYPE c LENGTH 30,
+    END OF ty_binding_mappings,
+
+    "! <p class="shorttext">Dataset Bindings</p>
+    "! These are scenario dataset bindings
+    BEGIN OF ty_bindings,
+      "! <p class="shorttext">Name</p>
+      "! This is the name of the binding
+      name      TYPE c LENGTH 30,
+      "! <p class="shorttext">Reference UUID</p>
+      "! This is a reference UUID
+      reference TYPE c LENGTH 36,
+    END OF ty_bindings,
+
+    "! <p class="shorttext">Signatures</p>
+    "! These are the scenario signatures
+    BEGIN OF ty_signatures,
+      "! <p class="shorttext">Signature Name</p>
+      "! This is the signature name
+      name           TYPE c LENGTH 30,
+      "! <p class="shorttext">Signature Type</p>
+      "! This is the signature type
+      signature_type TYPE c LENGTH 32,
+      "! <p class="shorttext">Description</p>
+      "! This is the description
+      description    TYPE c LENGTH 100,
+    END OF ty_signatures,
+
+    "! <p class="shorttext">Signature Fields</p>
+    "! These are the fields of signature
+    BEGIN OF ty_signature_fields,
+      "! <p class="shorttext">Name</p>
+      "! This is the signature input name
+      input_name     TYPE c LENGTH 30,
+      "! <p class="shorttext">Table or View Field Name</p>
+      "! This is a table or view field name
+      name           TYPE c LENGTH 30,
+      "! <p class="shorttext">Position</p>
+      "! This is the position of the field
+      field_position TYPE i,
+      "! <p class="shorttext">Storage</p>
+      "! This is the storage type of the field
+      storage        TYPE c LENGTH 100,
+      "! <p class="shorttext">Role of Field</p>
+      "! This is the role of field in an intelligent scenario (such as key, target)
+      type           TYPE c LENGTH 30,
+      "! <p class="shorttext">Data Management Type</p>
+      "! This is the type of data management
+      dm_type        TYPE c LENGTH 30,
+    END OF ty_signature_fields,
+
+    "! <p class="shorttext">Modelling Context</p>
+    "! This is the intelligent scenario Modelling context
+    BEGIN OF ty_modelling_context,
+      "! <p class="shorttext">Name</p>
+      "! This is the name of the modelling context
+      name        TYPE c LENGTH 30,
+      "! <p class="shorttext">UUID</p>
+      "! This is the modelling context UUID
+      guid        TYPE c LENGTH 36,
```

Comment:
As mentioned in our call. It would be great if we could avoid GUIDs in AFF. 

As far as I understood `INTM` links to `INTS` via this GUID. If the GUID is not necessary to be known by other objects, you could think of to use different GUIDS in different systems. Maybe?

Reply:
Actually, I've introduced 'Intelligent scenario' in the file format and displaying it in the ADT UI. But we cannot avoid the GUID as it's tightly integrated with our ISLM Management Fiori App. So, once INTS & INTM is created, it will be consumed for operation in the ISLM Management Fiori App. So, it should be created in the Original system and transported.
But I've renamed the field to 'modelling_context_id' as readable one.